tck/README.md

```diff
@@ -411,9 +369,40 @@ public class MySubscriberWhiteboxVerificationTest extends SubscriberWhiteboxVeri
 }
 ```
 
+### Subscriber Blackbox Verification
+
+Blackbox Verification does not require anything besides providing a `Subscriber` and `Publisher` instances to the TCK,
+at the expense of not being able to verify as much as the `SubscriberWhiteboxVerification`:
+
+```java
+package com.example.streams;
+
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+import org.reactivestreams.tck.SubscriberBlackboxVerification;
+import org.reactivestreams.tck.TestEnvironment;
+
+public class MySubscriberBlackboxVerificationTest extends SubscriberBlackboxVerification<Integer> {
+
+  public MySubscriberBlackboxVerificationTest() {
+    super(new TestEnvironment());
+  }
+
+  @Override
+  public Subscriber<Integer> createSubscriber() {
+    return new MySubscriber<Integer>();
+  }
+
+  @Override
+  public Integer createElement(int element) {
+    return element;
+  }
+}
+```
```

Comment:
Oh I see, you just moved it below. Also good idea, since we want to encourage using Whitebox.